From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Shelley Vohr <shelley.vohr@gmail.com>
Date: Tue, 12 Jan 2021 09:17:14 -0800
Subject: fix: remove outdated --experimental-wasm-bigint flag

The --experimental-wasm-bigint flag was removed in https://chromium-review.googlesource.com/c/v8/v8/+/2610965
but not yet from Node v14, as its version of V8 is not recent enough.

This patch can be removed as soon as we upgrade Node.js to a version of
V8 which contains the above CL.

diff --git a/test/wasi/test-return-on-exit.js b/test/wasi/test-return-on-exit.js
index 3f5d40c9ba7fb864767e9ac6c4b9a9f466d687e7..362d13b490a96f16029ba7edc0cda14db0feaab6 100644
--- a/test/wasi/test-return-on-exit.js
+++ b/test/wasi/test-return-on-exit.js
@@ -1,4 +1,4 @@
-// Flags: --experimental-wasi-unstable-preview1 --experimental-wasm-bigint
+// Flags: --experimental-wasi-unstable-preview1
 'use strict';
 const common = require('../common');
 const assert = require('assert');
diff --git a/test/wasi/test-wasi-not-started.js b/test/wasi/test-wasi-not-started.js
index ad13e6d711802b029c4b536f2ed8944484d821cf..14c8cdfddf2f5cc7c8219cc7810bbdcb56482905 100644
--- a/test/wasi/test-wasi-not-started.js
+++ b/test/wasi/test-wasi-not-started.js
@@ -29,7 +29,6 @@ if (process.argv[2] === 'wasi-child') {
 
   const child = cp.spawnSync(process.execPath, [
     '--experimental-wasi-unstable-preview1',
-    '--experimental-wasm-bigint',
     __filename,
     'wasi-child'
   ], {
diff --git a/test/wasi/test-wasi-stdio.js b/test/wasi/test-wasi-stdio.js
index 4abe3c1ad8ae0d7af7b57040fdc4b146931a2b15..647990064efb438e36b0c5ec9a3480338d09b2f9 100644
--- a/test/wasi/test-wasi-stdio.js
+++ b/test/wasi/test-wasi-stdio.js
@@ -1,4 +1,4 @@
-// Flags: --experimental-wasi-unstable-preview1 --experimental-wasm-bigint
+// Flags: --experimental-wasi-unstable-preview1
 'use strict';
 require('../common');
 const tmpdir = require('../common/tmpdir');
diff --git a/test/wasi/test-wasi-symlinks.js b/test/wasi/test-wasi-symlinks.js
index d1ec796125cb532e95e27562620312fdae40fac3..f619a8a2439a7d0c8624ffb31f1a83b87a9ad5ba 100644
--- a/test/wasi/test-wasi-symlinks.js
+++ b/test/wasi/test-wasi-symlinks.js
@@ -62,7 +62,6 @@ if (process.argv[2] === 'wasi-child') {
     const opts = { env: { ...process.env, NODE_DEBUG_NATIVE: 'wasi' } };
     const child = cp.spawnSync(process.execPath, [
       '--experimental-wasi-unstable-preview1',
-      '--experimental-wasm-bigint',
       __filename,
       'wasi-child',
       options.test,
diff --git a/test/wasi/test-wasi.js b/test/wasi/test-wasi.js
index b4c404e515cbb2a54eac4fd52eb92d8535b577d9..85543255b2ddf8f8ebd7f7893ea0fe0c60513b22 100644
--- a/test/wasi/test-wasi.js
+++ b/test/wasi/test-wasi.js
@@ -51,7 +51,6 @@ if (process.argv[2] === 'wasi-child') {
 
     const child = cp.spawnSync(process.execPath, [
       '--experimental-wasi-unstable-preview1',
-      '--experimental-wasm-bigint',
       __filename,
       'wasi-child',
       options.test
